### PR TITLE
[s] All the small things

### PR DESCRIPTION
**Pull request by Incoming5643**
Various bits and bobs I couldn't really place elsewhere

Unlimited clown and mime job slots

All admins gain +SOUND :cop: :ok_hand: 

Removes the MLP pull affecting the station name. I don't want people to see something wrong with the title because to some it might imply that that's the only joke to be found. Also some people would probably just rage quit for the day and we don't want that.

:rose: :rose: :rose:

### CODE REVIEW

File: code/modules/jobs/job_types/civilian.dm [hunk 1]
```diff
@@ -7,8 +7,8 @@ Clown
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = 1
-	spawn_positions = 1
+	total_positions = -1
+	spawn_positions = -1
```
Comment (Supermichael777) [inline code/modules/jobs/job_types/civilian.dm]: wont this result in a massive dog-pile?

Comment (Incoming5643) [inline code/modules/jobs/job_types/civilian.dm]: and?

**Comment by Durkel** [comment]
>Removes one of last year's April fools jokes where all sounds the admins tried to play would get modulated up/down to fuck with things.

Don't, this can be fucking hilarious and it's not like the midis get any worse anyway.

**Comment by Zilenan91** [comment]
Can you remove race blockers for all jobs for april fools?

**Comment by Incoming5643** [comment]
That's a config setting, if they want to turn that off they already can

**Comment by Incoming5643** [comment]
@Durkel midis don't, oggs however do

**Comment by Niknakflak** [comment]
>too busy to work on datum antags

>makes like 5 april fools threads instead


Priorities

**Comment by Zilenan91** [comment]
He needs a break every now and again, don't be an ass.

**Comment by Incoming5643** [comment]
Datum antag work is being inhibited by the fact that every time I try to start a meaningful task with it I get a overwhelming urge to jump out of a window. It's hard to get productive about something I don't enjoy doing, especially when I'm not getting any sort of compensation for it. What it could do for the game is pretty amazing, but it's hard to look out into the distance and gain motivation for a goal line so far away.

The memes flow easy, the meaningful stuff takes real disciple and effort; two things I'm currently hemorrhaging about as fast as I gain it. I was probably not the best person to try and do this, but with time I hope to get there.

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -234,6 +234,7 @@ var/list/admin_verbs_hideable = list(
 
 		var/rights = holder.rank.rights
 		verbs += admin_verbs_default
+		verbs += admin_verbs_sounds
```
Comment (duncathan) [inline code/modules/admin/admin_verbs.dm]: hnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnng
:ok_hand:

Comment (PKPenguin321) [inline code/modules/admin/admin_verbs.dm]: Dubstep and Shrek midis go go go

**Comment by Incoming5643** [comment]
Alright I put the old joke back in since people apparently really like it, it's practically april fools day, lets get this going

**Pull request merged**